Installation/CHANGES.md

```diff
@@ -455,7 +464,7 @@ Release date: November 2019
   - LLVM Clang version 8.0 or later (on Linux or MacOS), and
   - Apple Clang compiler versions 7.0.2 and 10.0.1 (on MacOS).
 - Since CGAL 4.9, CGAL can be used as a header-only library, with
-  dependencies. Since CGAL 5.0, that is now the default, unless
```

Comment:
unbreakable spaces, maybe ?

Reply:
> unbreakable spaces, maybe ?

Yes, of course that is unbreakable spaces. But the patch is wrong, and it should not modify those spaces. We already had that discussion in a previous PR.

Reply:
Yes, well it is not MY patch, and if unbreakable spaces become regular spaces, it usually is because of the writer's text editor, probably with automatic indentation. Of course it is wrong, but it is probably not wanted by the developer, and knowing it is unbreakable spaces changed by the editor might help fixing the problem.

Reply:
Sorry Maxime, it did not mean to go against you specifically. You are right that the patch is probably because of a dump text editor that is smart enough to recognize non-breaking spaces as spaces, but dump enough to convert them back to normal spaces.

@danston Try to see if you can convince your editor not to convert spaces. And have a look at the patch before using `git add` or `git commit`.

Reply:
Are you sure it comes from my commit? Honestly, I do not even remember ever opening this file when working on this branch. When I open it now, my editor shows me the correct text. Which commit is it?

Reply:
It is from one of my commits, but it has nothing to do in that branch, and has been fixed before. Isn't there a problem of merging, or isn't it based on the wrong branch  ?

Reply:
I think I based my branch on Sebastien's branch that is based on the student's branch and then I pushed it to Sebastien's remote.

Reply:
@sloriot  is this resolved?

Reply:
I think I close it because it is not a problem of this PR.